aten/src/ATen/native/TensorProperties.cpp

```diff
@@ -28,6 +31,20 @@
 
 namespace at::native {
 
+Tensor& _apply_cow_(Tensor& self) {
+  //at::functorch::TensorWrapper* wrapped = at::functorch::maybeGetTensorWrapper(self);
+  //TensorImpl* tensor_impl = wrapped ?
+  //  wrapped->value().unsafeGetTensorImpl() :
+  //  self.unsafeGetTensorImpl();
+
+  TensorImpl* tensor_impl = at::functorch::getBaseWrappedTensor(self).unsafeGetTensorImpl();
```

Comment:
I added this `getBaseWrappedTensor` because there's a unit test where `_apply_cow_` gets called on a tensor that is a `TensorWrapper`, which causes an error when trying to access the `TensorWrapper`'s storage. So I added `getBaseWrappedTensor` to obtain the innermost tensor. But I doubt that this is the right solution. I imagine the real fix must be to change how `at::_apply_cow_` dispatches `TensorWrapper`s